sleap_nn/training/model_trainer.py

```
@@ -463,17 +468,18 @@ def run_subprocess():
                     f"New dir is created and `.bin` files are saved in {self.bin_files_path}"
                 )
 
                 if not Path(self.bin_files_path).exists():
                     try:
                         Path(self.bin_files_path).mkdir(parents=True, exist_ok=True)
                     except OSError as e:
-                        raise OSError(
+                        logger.error(
                             f"Cannot create a new folder in {self.bin_files_path}. Check the permissions to the given Checkpoint directory. \n {e}"
                         )
+                        raise OSError()
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)